src/app/map/[id]/Tile/navigation.button.static.tsx

```diff
@@ -0,0 +1,69 @@
+import Link from "next/link";
```

Comment:
I think this file is unused and should be removed. Or maybe keep it (but I think the code in src/app/map/[id]/Tile/item.buttons.static.tsx is more up to date), create a src/app/map/[id]/Tile/change-expansion.button.static.tsx, and simplify the code in item.buttons.static.tsx to use import from these 2 files instead.

Also there are a lot of files in this Tile folder, I think you should reorganize them using 3 new folders: Base (containing base and content), Item (contain all "item related" tile files including the item buttons) and Auth (for auth tile files). Then add a barrel index.ts file to simplify imports.